frontend/web/src/app/services/customer/customer.service.ts

```diff
@@ -43,6 +45,28 @@ export class CustomerService {
       });
   }
 
+  getDigitalParts(id: number): Observable<DigitalPart[]> {
+    return this.client.get(this.endpoint + '/' + id + '/digital-parts')
+        .map((data) => {
+            const digitalParts: DigitalPart[] = [];
+            for (let i = 0; i < data.length; i++) {
+              digitalParts[i] = DigitalPart.create(data[i]);
+            }
+            return digitalParts;
+          });
+  }
+
+  getOrders(id: number): Observable<Order[]> {
+    return this.client.get(this.endpoint + '/' + id + '/orders')
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)